### PR TITLE
task-1 completed

### PR DESCRIPTION
**Pull request by sonalikari**

### CODE REVIEW

File: app/database.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+const { MongoClient } = require('mongodb');
+
+const uri = 'mongodb://localhost:27017';
```
Comment (anmolint) [inline app/database.js]: Do not hard-code the credentials use environment variable I see a dotenv dependency why are you not using it?

File: app/index.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+      await db.collection('UsersProfile').insertOne(userProfileData);
+      console.log('UserProfile inserted for user:', userId);
+    }
+  } catch (err) {
+    console.error('Error inserting users data:', err);
+  }
+}
+
+connectToMongoDB().then(() => {
```
Comment (anmolint) [inline app/index.js]: We can move this to async await from promise .then format also what would happen if error is thrown during promise execution How would you handle this?

File: app/sampleUsers.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+      {
+        firstName: 'Anmol',
+        lastName: 'Kumar',
+        email: 'anmol@example.com',
+        password: 'password5',
+        dob: '01-03-2001',
+        mobileNo: '9876543210'
+      },
+  ];
+
```
Comment (anmolint) [inline app/sampleUsers.js:43]: Please use .gitignore to not push node_modules these will make code base unnecessarily large

**Review by anmolint — COMMENTED** [review]
Please fix  and re create PR

File: .env
Comment (anmolint) [inline .env:1]: why do  we have .env here?

File: .gitignore
Comment (anmolint) [inline .gitignore:1]: it should have .env not .env/

**Review by anmolint — CHANGES_REQUESTED** [review]

File: .gitignore
Comment (sonalikari) [reply .gitignore:1]: ok

**Pull request merged**